.github/scripts/prepare-release-artifacts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+BINARY_NAME = "clio_server"
+
+GITHUB_ARTIFACTS = "$(ls)"
+
+for artifact_name in "${GITHUB_ARTIFACTS}; do
+    pushd "${artifact_name}" || exit 1
+    zip -r "../${artifact_name}.zip" ./${BINARY_NAME}
+    popd || exit 1
+
+    rm "${artifact_name}/${BINARY_NAME}"
+    rm -r "${artifact_name}"
+
+    sha256sum "./${artifact_name}.zip" > "./${artifact_name}.zip.sha256sum"
```

Comment:
Maybe instead we should sign binaries and provide signature files? But this could be done in a separate PR.